deployments/container/Dockerfile.ubi8

```diff
@@ -39,7 +39,7 @@ ARG VERSION="N/A"
 ARG GIT_COMMIT="unknown"
 RUN make PREFIX=/artifacts cmds
 
-FROM nvidia/${CUDA_IMAGE}:${CUDA_VERSION}-base-${BASE_DIST}
+FROM nvcr.io/nvidia/${CUDA_IMAGE}:${CUDA_VERSION}-base-${BASE_DIST}
```

Comment:
@cdesiniotis is this cuda-base image public on nvcr.io?

Reply:
Yes, it is public